lib/storage/acl.js

```diff
@@ -295,13 +418,55 @@ Acl.prototype.update = function(options, callback) {
 };
 
 /**
+ * Make a {module:storage/bucket} or {module:storage/file} object private.
+ *
+ * This is a short-hand method which will remove ACL permissions for the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePrivate
+ *
+ * @example
+ * myBucket.acl.makePrivate(function(err) {});
+ * myFile.acl.makePrivate(function(err) {});
+ */
+Acl.prototype.makePrivate = function(callback) {
+  this.delete({
+    scope: 'allUsers'
+  }, callback);
+};
+
+/**
+ * Make a {module:storage/bucket} or {module:storage/file} object publicly
+ * readable.
+ *
+ * This is a short-hand method which will grant "reader" permissions to the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePublic
+ *
+ * @example
+ * myBucket.acl.makePublic(function(err, aclObject) {});
```

Comment:
The examples have used `aclObject` since we added acl support. I tried to make the name overly descriptive just to help. Do you think we should switch them all to `acl`?